scripts/env-vars-setup

```diff
@@ -100,9 +100,9 @@ export PCS_CORS_WHITELIST=""
 
 :: URLs
 export PCS_AUTHENTICATION_WEBSERVICE_URL="http://127.0.0.1:9001/v1"
-export PCS_IOTHUBMANAGER_WEBSERVICE_URL="http://iothub-manager:9002/v1"
-export PCS_DEVICESIMULATION_WEBSERVICE_URL="http://device-simulation:9003/v1"
-export PCS_TELEMETRY_WEBSERVICE_URL="http://telemetry:9004/v1"
-export PCS_CONFIG_WEBSERVICE_URL="http://pcs-config:9005/v1"
-export PCS_STORAGEADAPTER_WEBSERVICE_URL="http://storage-adapter:9022/v1"
```

Comment:
The goal is to use the url name that is used by the docker containers. Why are we changing this?

Reply:
This file is not used by docker. Docker containers use values from "env.list" file. This file is for developer convenience to setup env vars globally. These variables define how each micro-services talk to each other.

Reply:
done